docs/agents/standard.rst

```diff
@@ -18,14 +18,13 @@ TableAgent
 | Table agents are the simplest form of RL Agents.
 | With experience, we build an action_value array (Q in literature).
 | Q(s,a) being the expected futur rewards given that the agent did the action a in the state s.
-| For that, they are composed of two main objects : Control and Evaluation
+| For that, they are composed of two main objects : :ref:`Control` and :ref:`Evaluation`.
 
 | A :class:`~learnrl.control.Control` object uses the action_value to determine the policy or behavior of the agent.
 | An :class:`~learnrl.evaluation.Evaluation` object predicts the expected rewards from an experience given a behavior.
 
-| By default, when building a :class:`~learnrl.agents.StandardAgent`, it will be a TableAgent (using a :class:`~learnrl.estimator.TableEstimator`) 
-| with eps-greedy control (see :ref:`Control`)
-| and QLearning evaluation (see :ref:`Evaluation`)
+| By default, when building a :class:`~learnrl.agents.StandardAgent`, it will not use DeepLearning (using a :class:`~learnrl.estimator.TableEstimator`) 
```

Comment:
Don't we miss an 's' in 'estimators' here ?

Reply:
Fixed it !

Reply:
yeah got it